README.md

```diff
@@ -29,6 +29,15 @@ Download the most recent `Monocraft.otf` file from the [Releases](https://github
 
 ### Mac
 
+#### Using homebrew
```

Comment:
Commands look good, only suggestions are to capitalize "Homebrew" in this header and reorder the subheadings so that "Manually" comes first since most people will probably be using that method.

Reply:
Done! :)